pkg/coordinator/internalhandlers.go

```diff
@@ -1,25 +1,32 @@
 package coordinator
 
 import (
+	"encoding/base64"
+	"encoding/json"
 	"log"
 
-	"github.com/giongto35/cloud-game/v2/pkg/config/worker"
+	api2 "github.com/giongto35/cloud-game/v2/pkg/api"
```

Comment:
Hate namespaces.

Reply:
Moved into cws.api.coordinator. I ported it from "refactor netcode" PR where I removed cws package, here it's better to do as I did.